cmd/containerd-shim-runc-v2/process/io_test.go

```diff
@@ -49,6 +50,25 @@ func TestNewBinaryIO(t *testing.T) {
 	}
 }
 
+func TestNewAttachableBinaryIO(t *testing.T) {
+	testutil.RequiresRoot(t)
+
+	ctx := namespaces.WithNamespace(context.Background(), "test")
+	uri, _ := url.Parse("binary:///bin/echo?test")
```

Comment:
how to integrate it from containerd side? the binary scheme is used by original `binary` one.

Reply:
good catch, fixed